.buildkite/scripts/run_models.sh

```diff
@@ -31,5 +31,9 @@ dbt run --target "$db" --full-refresh
 dbt test --target "$db"
 dbt run --vars '{shopify_timezone: "America/New_York", shopify_using_fulfillment_event: true, shopify_using_abandoned_checkout: false, shopify_using_metafield: false, shopify_using_discount_code_app: true, shopify_product_variant_media: true}' --target "$db" --full-refresh
 dbt test --target "$db"
+if [ "$db" = "bigquery" ]; then
+dbt run --vars '{shopify_collection_identifier: shopify_collection_bq_json_data, shopify_order_identifier: shopify_order_bq_json_data, shopify_transaction_identifier: shopify_transaction_bq_json_data}' --target "$db" --full-refresh
+dbt test --target "$db"
```

Comment:
We don't need to include the variables from line 35 with the `dbt test` right

Reply:
Correct, those variables are just used to set the upstream sources. The materialized models already have this applied and the test runs off those models.